packages/docs/developer-resources/contractkit/reference/SUMMARY.md

```diff
@@ -81,7 +81,6 @@
   * [ParticipationParameters](interfaces/_wrappers_governance_.participationparameters.md)
   * [ProposalMetadata](interfaces/_wrappers_governance_.proposalmetadata.md)
   * [ProposalRecord](interfaces/_wrappers_governance_.proposalrecord.md)
-  * [ProposalStageDurations](interfaces/_wrappers_governance_.proposalstagedurations.md)
```

Comment:
This no longer exists in the governance wrapper, I guess that's why the docs builder removed it.